lerobot/common/policies/diffusion/modeling_diffusion.py

```diff
@@ -165,7 +165,7 @@ def __init__(self, config: DiffusionConfig):
         num_images = len([k for k in config.input_shapes if k.startswith("observation.image")])
         self.unet = DiffusionConditionalUnet1d(
             config,
-            global_cond_dim=(config.output_shapes["action"][0] + self.rgb_encoder.feature_dim * num_images)
+            global_cond_dim=(config.input_shapes["observation.state"][0] + self.rgb_encoder.feature_dim * num_images)
```

Comment:
This change should fix the linting test

```suggestion
            global_cond_dim=(
                config.input_shapes["observation.state"][0] + self.rgb_encoder.feature_dim * num_images
            )
```

Reply:
@alexander-soare Actually, I just fixed linting problem on my forked repo.